src/main/docker/realm-config/jhipster-realm.json

```diff
@@ -2209,6 +2209,6 @@
     "clientSessionMaxLifespan": "0",
     "clientOfflineSessionIdleTimeout": "0"
   },
-  "keycloakVersion": "12.0.4",
+  "keycloakVersion": "15.0.2",
```

Comment:
though it's not correct update, I think we can got ahead and handle along upgrade ticket

Reply:
So, I leave as it is